src/main/java/Deadline.java

```diff
@@ -1,14 +1,14 @@
 public class Deadline extends Todo {
     protected String endDate;
 
-    public Deadline(String description, String endDate){
+    public Deadline(String description, String endDate) {
         super(description);
         this.endDate = endDate;
         this.tag = "D";
     }
 
     @Override
-    public String toString(){
+    public String toString() {
```

Comment:
Good job on adding the spacing after the method name!

Reply:
Thank you!